app/controllers/orders_controller.rb

```diff
@@ -225,16 +233,18 @@ def update_or_purchase
   def update
     load_statuses
     params[:order_datetime] = build_order_date if acting_as?
+
     @order.transaction do
+      @order.assign_attributes(order_params)
```

Comment:
I'm not sure if there's something we can do easily to avoid the code climate warning about attribute sanitization. I wonder if it's because there's a return case where it's `nil` as opposed to a `ActionController::Parameters`.